### DISCUSSION TITLE
How can I let the user type something into a form field?

### DISCUSSION THREAD

**Discussion by dhope1106** [gilbarbara/react-joyride#790] [Q&A]
First, I love the project, it's very close to what I need.

During the tour, I would like to allow my user to create a new "entry". The first step is to type in the name of the entry, but I can't seem to get the focus on the text field. Can this be done?

**Comment by dhope1106** [comment]
I understand from reading the document that not allowing keyboard entry is the expected behavior. I'm just hoping there is a workaround. Or possibly a pointer to the place in the code where this is enforced.

**Reply by jasonhutton-sd** [reply]
Did you try using `spotlightClicks: true` for that step?

I am having a similar issue while using that option for the step.
